extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -261,44 +261,28 @@ export class DataTierApplicationWizard {
 		const service = await DataTierApplicationWizard.getService(msSqlProvider);
 		const ownerUri = await azdata.connection.getUriForConnection(this.model.server.connectionId);
 
-		const result = await service.deployDacpac(this.model.filePath, this.model.database, this.model.upgradeExisting, ownerUri, azdata.TaskExecutionMode.execute);
-		if (!result || !result.success) {
-			vscode.window.showErrorMessage(
-				localize('alertData.deployErrorMessage', "Deploy failed '{0}'", result.errorMessage ? result.errorMessage : 'Unknown'));
-		}
+		service.deployDacpac(this.model.filePath, this.model.database, this.model.upgradeExisting, ownerUri, azdata.TaskExecutionMode.execute);
```

Comment:
So was this resulting in duplicate errors or something? I'm confused why it matters that the wizard closes before the results return - the error message being displayed here should still happen even if the wizard isn't actually visible at the moment. 

If the expectation is just to not have unnecessary errors displayed then I'd suggest you keep the awaits, otherwise as Alan said you're left with dangling promises which should really be handled in some way.

Reply:
Added back the awaits after discussing since they aren't blocking anything.

Before this PR https://github.com/microsoft/sqltoolsservice/pull/905 the request from the SqlToolsService would return after starting the operation, so errors from the operation wouldn't show up here and so they are unnecessary. The real error would just show in the task viewlet.